### ISSUE TITLE
训练过程中IoU评估结果很高，用val评估的时候结果非常低

### ISSUE THREAD

**Issue by lxlhh** [PaddlePaddle/PaddleSeg#1718]
paddle 2.1.1
paddeseg 2.3.0

训练过程中，验证集的评估很正常，平均IoU 在0.8左右，但是相同iter的模型，用val.py直接测试验证集，结果很差，IoU只有0.3，用predict.py可视化分割结果，也很差

**Comment by LutaoChu** [comment]
是不是图像预处理没有对齐呀

**Comment by lxlhh** [comment]
val.py和predict.py也是用val_dataset的transforms进行数据处理，跟训练过程中的eval应该是一样的才对呀
### 
train_dataset:
  type: Dataset  
  transforms:
    - type: Resize
      target_size: [256, 256]
    - type: RandomHorizontalFlip
    - type: RandomDistort
      brightness_range: 0.4
      contrast_range: 0.4
      saturation_range: 0.4
    - type: Normalize
  mode: train
### 
val_dataset:
  type: Dataset
  transforms:
    - type: Resize
      target_size: [256, 256]
    - type: Normalize
  mode: val
###

**Comment by LutaoChu** [comment]
是的，训练过程评估调用的是core.evaluate()，val.py调用的也是core.evaluate，理论上应该是要一样才对
建议打印一下两者调用core.evaluate()时的参数，看看是不是一样吧
